package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "mimic-fn",
-	"version": "3.0.0",
+	"version": "3.0.1",
```

Comment:
The release is made separately from this PR, could you please remove this version bump?